index.html

```diff
@@ -1,19 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/nav.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/footer.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img class="logo" src="img/karma-logo.svg" alt="logo" />
+        <img
+          class="menu-burger"
+          src="img/menu-hamburger.svg"
+          alt="menu-burger"
+        />
+        <ul>
+          <li>
+            <a href="../index.html" class="link-strong">Meet Karma</a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li><a href="../form.html" target="_blank">Store</a></li>
```

Comment:
* `./` means element is in the same folder where `../` means element is in the parent folder. Both `index.html` and `form.html` shall be at the same folder
* consider renaming `form.html` to `store.html`?